Makefile

```diff
@@ -16,8 +16,9 @@ deps:
 ## fmt: Formats only *.go (excluding *.pb.go *pb_test.go). Runs `gofmt & goimports` internally.
 fmt: sort-imports
 	@find . -name '*.go' -type f -not -path "*.git*" -not -name '*.pb.go' -not -name '*pb_test.go' | xargs gofmt -w -s
-	@find . -name '*.go' -type f -not -path "*.git*"  -not -name '*.pb.go' -not -name '*pb_test.go' | xargs goimports -w -local github.com/celestiaorg
+	@find . -name '*.go' -type f -not -path "*.git*" -not -name '*.pb.go' -not -name '*pb_test.go' | xargs goimports -w -local github.com/celestiaorg
 	@go mod tidy
+	@gofumpt -w -extra .
 	@cfmt -w -m=100 ./...
```

Comment:
Should we drop this in favour of `lll` which is a part of `golangci-lint` run?

Reply:
Yes pls, could we just integrate golangci-lint here? basically copy the current node's setup

Reply:
Done. I'm not going to add `golangci-lint` to CI now, would like to add in a separate PR.